config/settings/keycloak.py

```
@@ -0,0 +1,158 @@
+
+# The Keycloak Realm that contains your users, roles, groups, and clients.
+KEYCLOAK_REALM = "EndoregDb"
+
+# The specific Keycloak "Client" (OIDC client/app) your Django app uses.
+# It must exist in the realm, with Standard Flow enabled, proper redirect URIs, and Web Origins configured.
+OIDC_RP_CLIENT_ID = "endoregdb-api"
+
+# The client secret for the OIDC client above. This authenticates Django to Keycloak during token exchange.
+# ⚠️ Best practice: load this from environment variables, not hardcoded in git.
+# e.g. from endoreg_db.config.env import env_str; OIDC_RP_CLIENT_SECRET = env_str("OIDC_RP_CLIENT_SECRET", "")
+OIDC_RP_CLIENT_SECRET = "18jcfrebu8kw2sbWE0kJ3h9OdMfI3E2S"
```

Ruff (0.14.5)

Possible hardcoded password assigned to: "OIDC_RP_CLIENT_SECRET"

(S105)